typescript-dto/publish.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env sh
+
+#get the latest version of @eclipse-che/api
+api_version=$(yarn -s info @eclipse-che/api version)
```

Comment:
we should have

```
set -e
set -u
```

 as well

should we first check that yarn tool is installed as well ?

Reply:
I added 
```
set -e
set -u
```
No, as that script called from docker which already have `yarn` https://github.com/eclipse/che/blob/b818fccea3496f85579ad1affaf880674bb718f1/typescript-dto/Dockerfile#L25